config_examples/prometheus.yaml

```diff
@@ -30,5 +31,5 @@ service:
   pipelines:
     metrics:
       receivers: [prometheus]
-      processors: [cumulativetodelta]
+      processors: [metricstarttime, cumulativetodelta]
```

Comment:
shouldn't we add the processor to all the configs, that are using cumulativetodelta processor? The configs are here https://github.com/Dynatrace/dynatrace-otel-collector/commit/aa0989fef753424fdee85c3721cde0aba2d77187

Also, should we consider a test for this component?

Reply:
>shouldn't we add the processor to all the configs, that are using cumulativetodelta processor?

We only need this processor in cases where the start timestamp has not been set. For us this should only be the Prometheus receiver, which is the only component that was affected by the recent change to not set starting timestamps.

>Also, should we consider a test for this component?

I'm open to adding some tests. I didn't add them in this PR in the interest of time, but if I have time this week I can add some.

Reply:
All good, I would prefer adding at least some simple sanity tests, that the starttime is actually set. We can reuse probably the existing integration test and just check that the metrics have startime set (to whatever value). It can potentially be just an additional assert [here](https://github.com/Dynatrace/dynatrace-otel-collector/blob/main/internal/testbed/integration/prometheus/e2e_test.go#L88), but if you prefer to add it in a separate PR, fine by me.